Bootstrap/Update.php

```diff
@@ -466,4 +467,47 @@ private function createRemoteToLocalCategoriesTable()
             }
         }
     }
+
+    private function recreateRemoteCategoriesAndProductAssignments()
+    {
+        if (version_compare($this->version, '1.1.4', '<=')) {
+            try {
+                $result = $this->db->query('SELECT `article_id`, `category` FROM `s_plugin_connect_items` WHERE exported IS NULL');
```

Comment:
better use ```shop_id``` column. This is the indicator to show you that some product is imported from connect